string name
---
bool success
string message
mission_control/MissionData data
